src/main/java/unicash/commons/enums/CommandType.java

```diff
@@ -398,7 +398,7 @@ public boolean isCommandWord(String commandWord) {
     }
 
     public String getMessageFailure() {
-        return "Command not recognised. Try using the command " + getMainCommandWord()
+        return "Command not recognised. Try using the command " + getCommandWords()
```

Comment:
Just wondering, which commands are affected by this change btw? I thought it would be `list` but `list` seems to already show all command words in its error message.

Reply:
`getMessageFailure()` is used in 4 command classes - `Clear`, `Edit`, `List` and `Reset`. However, this implementation can be overridden (read more about it [here](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/133)), which is the case for `Clear`, `Edit` and `Reset`.

This refactor is thus meant to only affect `List`, since it displays all command words in its error message, so I updated the `CommandType` enum to reflect that, in order to prevent any accidental message changes (which would violate feature freeze)

Hope this clears up things!

Reply:
Nice, thanks for explaining